src/pretix/control/templates/pretixcontrol/pdf/webfonts.css

```diff
@@ -14,7 +15,7 @@
 
 {% for family, styles in fonts.items %}
 {% for style, formats in styles.items %}
-{% if "sample" not in style %}
+{% if "sample" not in style and "pdf_only" not in style %}
```

Comment:
Well, the plugin API does allow to have `style["pdf_only"] = False` in which case this condition would be wrong, wouldn't it? :thinking:

Reply:
I did check it for a font that doesn't have `pdf_only`, that has it set to `True` and one that has it set to `False` - in either case it worked for me...

But perhaps we're looking at two different things here?

Reply:
As per direct discussion: Works as intended.